contracts/levels/Switch.sol

```diff
@@ -0,0 +1,36 @@
+pragma solidity ^0.4.24;
+
+contract Switch {

```

Comment:
This level idea's great!

Reply:
Hey, 
Thanks. This actually came from a contract that was to go live. Happy to modify it, to better suit Ethernaut.